flax/linen/module.py

```diff
@@ -224,6 +224,35 @@ def compact(fun: _CallableT) -> _CallableT:
   return fun
 
 
+def nowrap(fun: _CallableT) -> _CallableT:
+  """Marks the given module method as a helper method that needn't be wrapped.
+
+  Methods wrapped in @nowrap are private helper methods that needn't be wrapped
+  with the state handler or a separate named_call transform.
+
+  This is needed in two concrete instances:
+   - if you have a helper method that returns Modules or Variables to prevent
+     it from being functionalized by named_call. (Functionalized methods
+     can't return Modules/Variables.)
+   - if you're subclassing a method like param and don't want this overriden
```

Comment:
Another case:
- If you want a method to be callable from an unbound Module (e.g.: a function of construction of arguments that doesn't depend on params/RNGs)

Reply:
good addition, added it.